### PR TITLE
RET-508

### PR DESCRIPTION
**Pull request by brendanmchugh**
### JIRA link ###

https://tools.hmcts.net/jira/browse/RET-508

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/test/routes/single-or-multiple-claim.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { expect } from 'chai';
+import request from 'supertest';
+
+import { app } from '../../main/app';
+
+const expectedLegacyURL = 'https://employmenttribunals.service.gov.uk/apply';
```
Comment (marcjudgey) [inline src/test/routes/single-or-multiple-claim.ts:6]: Please extract to a global constant

Comment (brendanmchugh) [reply src/test/routes/single-or-multiple-claim.ts]: Thanks Marc, I've added a contants.ts file within the main directory and included the URL.
https://github.com/hmcts/et-sya-frontend/blob/RET-508/src/main/constants.ts

File: src/main/constants.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const URLS = {
```
[the remark this answers is not in the document]
Comment (brendanmchugh) [reply src/main/constants.ts]: I've moved contants.ts to definitions/contants.ts

File: src/main/constants.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+  LEGACY_ET1: 'https://employmenttribunals.service.gov.uk/apply',
+} as const;
```
[the remark this answers is not in the document]
Comment (brendanmchugh) [reply src/main/constants.ts]: As discussed there are some benefits to sharing constants in this way.
 https://medium.com/codex/how-to-share-constants-in-typescript-project-8f76a2e40352

**Review by marcjudgey — APPROVED** [review]

**Review by tautvydas-cerniauskas — APPROVED** [review]

**Review by jsinghm13 — APPROVED** [review]
Approving on behalf of Marc

**Pull request merged**